tests/testing.py

```diff
@@ -0,0 +1,312 @@
+from cosmpy.aerial.wallet import LocalWallet
+from cosmpy.aerial.tx import Transaction
+from cosmpy.crypto.keypairs import PrivateKey
+from cosmpy.crypto.address import Address
+from cosmpy.protos.cosmos.gov.v1beta1 import tx_pb2 as gov_tx, gov_pb2, query_pb2 as gov_query, query_pb2_grpc
+from cosmpy.protos.cosmos.base.v1beta1 import coin_pb2
+from cosmpy.protos.cosmwasm.wasm.v1 import tx_pb2 as wasm_tx, query_pb2 as wasm_query
+from cosmpy.aerial.contract import LedgerContract
+from bip_utils import Bip39SeedGenerator, Bip44, Bip44Coins
+from cosmpy.aerial.client import LedgerClient, NetworkConfig, utils
+from google.protobuf import any_pb2
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport
+import grpc, json, time, unittest, psycopg, os, requests
+unittest.TestLoader.sortTestMethodsUsing = lambda *args: 1
+
+
+class Base(unittest.TestCase):
+    delegator_wallet = None
+    delegator_address = None
+
+    validator_wallet = None
+    validator_address = None
+    validator_operator_address = None
+
+    ledger_client = None
+    db = None
+    db_cursor = None
+    gql_client = None
+
+    @classmethod
+    def setUpClass(cls):
+        validator_mnemonic = "nut grocery slice visit barrel peanut tumble patch slim logic install evidence fiction shield rich brown around arrest fresh position animal butter forget cost"
+        validator_seed_bytes = Bip39SeedGenerator(validator_mnemonic).Generate()
+        validator_bip44_def_ctx = Bip44.FromSeed(validator_seed_bytes, Bip44Coins.COSMOS).DeriveDefaultPath()
+        cls.validator_wallet = LocalWallet(PrivateKey(validator_bip44_def_ctx.PrivateKey().Raw().ToBytes()))
```

Comment:
Maybe we can factor these out to a helper for now. I think this would improve maintainability and readability.